app/models/contact-fragment.js

```diff
@@ -1,11 +1,10 @@
-import { attr } from '@ember-data/model';
-import Fragment from 'ember-data-model-fragments/fragment';
-import { computed } from '@ember/object';
+import{ attr } from '@ember-data/model';
 import { validator, buildValidations } from 'ember-cp-validations';
+import { computed } from '@ember/object';
+import Fragment from 'ember-data-model-fragments/fragment';
 
 const Validations = buildValidations({
   email: [
-    validator('presence', true),
```

Comment:
For a service contact, the email address is optional, but this change would make it optional for _all_ contact types. Is this the intended/desired behaviour? I did a quick scan of the docs but I couldn't see one way or the other, hopefully @MaryHirsch can confirm.

Reply:
I think that would be bad because this info is synced to our mailing lists and would useless without the email address. The email address should be mandatory and the name optional.

Reply:
@digitaldogsbody We have validation https://github.com/datacite/bracco/blob/3a6bee792f572300a6b98f273e02cc1c39fc4b4e/app/models/provider.js#L296 for email

Reply:
I think that's a computed attribute rather than a validator - and I don't see it being called anywhere in the block that handles contact validations: https://github.com/datacite/bracco/blob/3a6bee792f572300a6b98f273e02cc1c39fc4b4e/app/models/provider.js#L146-L208

Reply:
@digitaldogsbody I had another look with Suzzane and where we are validating the presence of the email we are using the `contact` model, `providers/:id/contacts/new`

Reply:
Ok great - I've had a quick look through the code and I think I am understanding how it works. Sorry for all the questions - I haven't really had the ability to pull the changes and test/deepdive locally due to commitments at the face to face EOSC meeting, so I am just trying to track it all through reading the code on Github!